app/helpers/drafts_helper.rb

```diff
@@ -543,15 +526,10 @@ module DraftsHelper
   URLContentTypeMap = url_content_type_map.deep_dup
   # UMM-S v1.1 added TOOL to GET SERVICE subtypes
   umm_s_url_content_type_map = url_content_type_map_for_umm_s.deep_dup
-  umm_s_url_content_type_map['DistributionURL']['types']['GET SERVICE']['subtypes'] << ['Tool','TOOL']
-  umm_s_url_content_type_map['DistributionURL']['types']['GET SERVICE']['subtypes'].sort!
   UMMSURLContentTypeMap = umm_s_url_content_type_map
   URLContentTypeOptions = [
     ['Collection URL', 'CollectionURL'],
     ['Publication URL', 'PublicationURL'],
-    ['Data Center URL', 'DataCenterURL'],
```

Comment:
This change will be reverted and a new constant created to capture the UMM-S list created in the next commit